bin/install.py

```diff
@@ -89,6 +97,19 @@ def download_and_install(url, target):
         print("Couldn't download %s, please try again." % url)
         sys.exit(1)
 
+def download_and_install_template(url, target):
```

Comment:
I think you could reuse the existing `download_and_install` and then modify the template once on disk (inside your `replace_template_variables`) instead of introducing socket handling and new urllib code

Reply:
This is a good idea :+1: